### PR TITLE
Allow switch to Simple from Step Policy

### PR DESCRIPTION
**Pull request by beckje01**
For the AWS Autoscaling policies allow switching to simple from step
policy.

### CODE REVIEW

**Comment by robzienert** [comment]
👍 Needs rebase.

File: app/scripts/modules/amazon/serverGroup/details/scalingPolicy/upsert/upsertScalingPolicy.modal.html [hunk 1]
```diff
@@ -32,7 +32,17 @@ <h4 class="text-center">Select a metric</h4>
           <aws-simple-policy-action command="ctrl.command"
                                     view-state="ctrl.viewState"></aws-simple-policy-action>
         </div>
+
```
Comment (icfantv) [inline app/scripts/modules/amazon/serverGroup/details/scalingPolicy/upsert/upsertScalingPolicy.modal.html]: whitespace

File: app/scripts/modules/amazon/serverGroup/details/scalingPolicy/upsert/upsertScalingPolicy.modal.html [hunk 2]
```diff
@@ -32,7 +32,17 @@ <h4 class="text-center">Select a metric</h4>
...
         <div ng-if="ctrl.command.step">
+
```
Comment (icfantv) [inline app/scripts/modules/amazon/serverGroup/details/scalingPolicy/upsert/upsertScalingPolicy.modal.html]: ditto

File: app/scripts/modules/amazon/serverGroup/details/scalingPolicy/upsert/upsertScalingPolicy.modal.html [hunk 3]
```diff
@@ -32,7 +32,17 @@ <h4 class="text-center">Select a metric</h4>
...
+          <div class="row">
+            <div class="col-md-10 col-md-offset-1">
+              <p>
+                <strong>Switch to a <a href ng-click="ctrl.switchMode()">simple policy</a>.</strong>
+              </p>
+            </div>
+          </div>
+
```
Comment (icfantv) [inline app/scripts/modules/amazon/serverGroup/details/scalingPolicy/upsert/upsertScalingPolicy.modal.html]: and here too

**Comment by robzienert** [comment]
After talking with the team, it turns out that simple scaling policies are deprecated within AWS, which is the reason why we support converting _to_ stepped scaling policies and not _from_ them. Closing, as not making this capability is a conscious decision. Should you need additional variability beyond what step policies provide, custom cloudwatch metric(s) for your individual applications to scale off of should do the trick. Happy to riff with you on Slack if you need help coming up with something.

**Pull request closed without merging**